### ISSUE TITLE
Do a release of updating lib

### ISSUE THREAD

**Issue by rufuspollock** [frictionlessdata/datapackage-java#37]
With lib now updated with appropriate open source json parser let's do a release so downstream can pull. Would be relevant for OpenRefine see e.g. https://github.com/OpenRefine/OpenRefine/issues/778#issuecomment-647021107

@roll would you be able to lead on this?

/cc @lwinfree

**Comment by roll** [comment]
Hi @iSnow, could you please release?

**Comment by iSnow** [comment]
Do we want to release manually or make it part of the CI pipeline? 

I can prepare the steps, but it would be good if you both read through:
- https://medium.com/@nmauti/publishing-a-project-on-maven-central-8106393db2c3
- https://dzone.com/articles/publish-your-artifacts-to-maven-central

since there are a couple of steps necessary that need the handling of credentials. It would be good if you have an email address that I can use for the Jira-account. I'd mail you the password to the Jira and the SSH cert but you absolutely must safekeep this on your side. Please don't come back in a year or five and ask me about the credentials.

**Comment by roll** [comment]
Thanks @iSnow, I'll set it up. I had forgotten about #25

**Comment by roll** [comment]
> It would be good if you have an email address that I can use for the Jira-account. I'd mail you the password to the Jira and the SSH cert but you absolutely must safekeep this on your side. Please don't come back in a year or five and ask me about the credentials.

Sure, we use the OKFN account usually for package registries so I just need to prepare all this stuff. It seems it requires wrapping my head around all this stuff for Java so I'll be on it in a week or so.

**Comment by iSnow** [comment]
@roll this might help: 

- https://blog.sonatype.com/2010/01/how-to-generate-pgp-signatures-with-maven/
- https://dzone.com/articles/publish-your-artifacts-to-maven-central
- https://medium.com/@nmauti/publishing-a-project-on-maven-central-8106393db2c3

**Comment by roll** [comment]
I've checked this guide - https://dzone.com/articles/publish-your-artifacts-to-maven-central - and it feels really overwhelming for a person who has never touched any Java code. It seems I need to set up the whole java dev environment and then do a lot of other steps I'm not sure about. As a last resort, I can, of course, do it but just wondering if there are simpler options.

@rufuspollock, maybe there is a person at Datopian who has Java experience and can create a frictionless account on Maven and release?

OR

@iSnow if you don't mind helping with it maybe you can point our what exact credentials I need to create so you can setup a release via Travis? I found these articles:
- https://github.com/stefanbirkner/travis-deploy-to-maven-central
- https://blog.travis-ci.com/2017-03-30-deploy-maven-travis-ci-packagecloud/

**Comment by roll** [comment]
It seems to release to https://packagecloud.io/ is much easier and there is a Travis automation for it but I'm not sure this package registry is what is needed for OpenRefine

**Comment by sniffertine** [comment]
Hi all. I use the library through jitpack. maybe a tag in this repo would be enough to have a "release"? 

With a tag, you can use the tag name as version with jitpack. In gradle, this would look like this: 
`implementation('com.github.frictionlessdata:datapackage-java:<tag-name>')`

**Comment by iSnow** [comment]
> Hi all. I use the library through jitpack. maybe a tag in this repo would be enough to have a "release"?

Good idea, I tagged this as "0.1.1". Ultimately, I'd like to get it on mvnrepository, but this will do for the time

BTW, I am looking for contributors, if you feel like you'd like to do some things better, please do submit tickets or PRs.

**Comment by sniffertine** [comment]
Cool. I will quickly integrate the tagged version here. 

And yes, I have a suggestion that might me worth at least a ticket. The datapackage files require to be on the filesystem, but my JVM integration here (https://github.com/datasets/unece-units-of-measure/pull/3) uses the datapackage from inside a .jar file. Currently, I copy the files from inside the jar into a temp folder before I am able to process the datapackage.
